theme/theme_test.go

```diff
@@ -1,17 +1,13 @@
 package theme
 
 import (
+	"image/color"
 	"testing"
 
-	"fyne.io/fyne"
 	"github.com/stretchr/testify/assert"
-)
 
-// TODO figure how to test the default theme if we are messing with it...
-//func TestThemeDefault(t *testing.T) {
-//	loaded := fyne.GlobalSettings().Theme()
-//	assert.NotEqual(t, lightBackground, loaded.BackgroundColor())
-//}
+	"fyne.io/fyne"
```

Comment:
I thought we wanted to have Fyne imports before 3rd party imports, didn’t we?

Reply:
I can't remember. Since switching to goimports the IDE wants it the other way.
Reverting this for the PR, but probably need to find a way for lint to ensure we are consistent.